### PR TITLE
CSI-3412 add device buffers flush

### PR DESCRIPTION
**Pull request by oriyarde**
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/5/html/online_storage_reconfiguration_guide/removing_devices

### CODE REVIEW

File: node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go [hunk 1]
```diff
@@ -69,7 +70,8 @@ const (
 	WaitForMpathWaitIntervalSec = 1
 	FC_HOST_SYSFS_PATH          = "/sys/class/fc_remote_ports/rport-*/port_name"
 	IscsiHostRexExPath          = "/sys/class/iscsi_host/host*/device/session*/iscsi_session/session*/targetname"
-	MpathdSeparator             = ","
+	blockdevCmd                 = "blockdev"
+	mpathdSeparator             = ","
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go]: but there is upper camel and lowe camel, its not really aligned IMO..

[the remark this answers is not in the document]
Comment (oriyarde) [reply node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go]: well, [that's just like, your opinion man](https://knowyourmeme.com/memes/thats-just-like-your-opinion-man)
https://tour.golang.org/basics/3

[the remark this answers is not in the document]
Comment (ArbelNathan) [reply node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go]: well, man, I cant see where is the uppercased consts are exported..

[the remark this answers is not in the document]
Comment (oriyarde) [reply node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go]: those that are not being exported indeed should be lowered, but it's not a matter of convention

[the remark this answers is not in the document]
Comment (oriyarde) [reply node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go]: CSI-3511

**Review by ArbelNathan — DISMISSED** [review]
LGTM

**Review by roysahar — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**